src/components/movies/MovieCard.js

```diff
@@ -551,6 +551,7 @@ function MovieCard({
                     size="large"
                     precision={0.5}
                     emptyIcon={<StarBorderIcon fontSize="inherit" />}
+                    // eslint-disable-next-line react/jsx-no-duplicate-props
```

Comment:
remove?

Reply:
this is in place to stop a duplicate prop error in VS code